### DISCUSSION TITLE
MPD to player change

### DISCUSSION THREAD

**Discussion by ghost** [rern/rAudio#267] [Fixed Issues]
@rern: Installed uodate 20210327 on rAudio 1 succefully.
Small omission detected. When clicking on the settings icon (right top cormer) "Player can be selected". But in the following screen the menybar at the bottom still shows "MPD". This is also ion the menu-items "Features", "Networks" and "System". Clicking "MPD" causes a flashing logo as this icon is linking to a non-axisting webpage.
Thanks, regards

**Comment by klassikmann** [comment]
Resolved in **rAudio 1 • 20210328**.
